elements/pl-rich-text-editor/pl-rich-text-editor.js

```diff
@@ -63,7 +63,10 @@ class MathFormula extends Embed {
 
   static updateNode(node, value) {
     MathJax.startup.promise.then(async () => {
-      let html = await MathJax.tex2chtmlPromise(value);
+      let html =
+        outputComponent === 'output/chtml'
```

Comment:
I reallllly don't want to rely on a bespoke global variable here. Could we instead inspect MathJax's configuration? `MathJax` is at least a standard and well-known global, and it looks like `MathJax.config.loader.load` can be checked for `output/chtml` or `output/svg`.

Reply:
Sure, I can do that. I realize this was completed in a rush when I was tired, so it can certainly use a bit more care.

Reply:
Resolved in 4c7a53c in a more elegant way, using the existence of the function itself as a signal.